schema/ordered_sets.js

```diff
@@ -14,6 +14,12 @@ const OrderedSets = {
       type: GraphQLBoolean,
       defaultValue: true,
     },
+    page: {
+      type: GraphQLInt,
+    },
+    size: {
+      type: GraphQLInt,
+    },
```

Comment:
Probably a good idea to set default values, I'm not sure if this is backwards compatible without it 👍

Reply:
Oh good call, I'll add that.